components/style/values/specified/box.rs

```diff
@@ -216,6 +216,12 @@ impl Display {
             other => other,
         }
     }
+
+    /// Returns true if the value is `None`
+    #[inline]
+    pub fn is_none(&self) -> bool {
```

Comment:
Perhaps controversial to add this? I felt like it made the code read a bit nicer, and it avoided an extra `use` declaration for the `Display` type, but I don't mind much either way. I can see how adding these sorts of methods all the time could get out of hand. (Note that it also reduced the implementation of `ElementStyles::is_display_none()`.)

Reply:
Looks fine to me.